subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r51/TaskExecutionResultCrossVersionSpec.groovy

```diff
@@ -124,16 +125,24 @@ class TaskExecutionResultCrossVersionSpec extends ToolingApiSpecification {
     def "reports incremental task as incremental"() {
         given:
         file('src').mkdir()
+        def supportsInputChanges = targetVersion > GradleVersion.version("5.4")
```

Comment:
Nit: I think extracting the parameter type as a variable (and maybe the annotation, too) would be nicer.

Reply:
Done.